empire/server/server.py

```diff
@@ -70,7 +71,15 @@ def setup_logging(args):
 INVOKE_OBFS_SRC_DIR_BASE = os.path.join(
     os.path.dirname(__file__), "data/Invoke-Obfuscation"
 )
-INVOKE_OBFS_DST_DIR_BASE = "/usr/local/share/powershell/Modules/Invoke-Obfuscation"
```

Comment:
This is the directory path also used in `install.sh` and `Dockerfile`. Can you update it there as well?

Reply:
Do you mean I must apply this dir change also on `Dockerfile` and `install.sh` files?

Reply:
I did some digging and it looks like since at least [v4.3.3](https://github.com/BC-SECURITY/Empire/compare/v4.2.0...v4.3.3#diff-75f581b062dfa810870551d8753a1dd263c7ad00254f301a62cb261c25391aceR92), `Invoke-Obfuscation` has been invoked directly from the `empire/server/data` directory and the `/usr/local/share/powershell/Modules/Invoke-Obfuscation` files are not touched at all.

So I think you would be safe to just delete everything in the server code relating to `INVOKE_OBFS_DST_DIR_BASE`.

You can also delete the lines in [install.sh](https://github.com/BC-SECURITY/Empire/blob/main/setup/install.sh#L45), [Dockerfile](https://github.com/BC-SECURITY/Empire/blob/main/Dockerfile#L63), and [docker](https://github.com/BC-SECURITY/Empire/blob/main/.github/cst-config-docker.yaml#L51) [tests](https://github.com/BC-SECURITY/Empire/blob/main/.github/install_tests/cst-config-install-base.yaml#L74) .

Reply:
In `Dockerfile`, in
```
RUN mkdir -p /usr/local/share/powershell/Modules && \
    cp -r ./empire/server/data/Invoke-Obfuscation /usr/local/share/powershell/Modules && \
    rm -rf /empire/empire/server/data/empire*
```
should I remove also `rm -rf /empire/empire/server/data/empire*`?

Reply:
@vinnybod In `Dockerfile`, in
```
RUN mkdir -p /usr/local/share/powershell/Modules && \
    cp -r ./empire/server/data/Invoke-Obfuscation /usr/local/share/powershell/Modules && \
    rm -rf /empire/empire/server/data/empire*
```
should I remove also `rm -rf /empire/empire/server/data/empire*`?

Reply:
Yes, keep that for now